rental.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+require './book'
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
```

Comment:
- [ ] This should be `book.rentals`, not `book.rental` based on what you defined in the book class